06_deploy_masters.sh

```diff
@@ -29,8 +29,7 @@ cp ocp/master.ign configdrive/openstack/latest/user_data
 for node in $(jq -r .nodes[].name $instack); do
 
   # FIXME(shardy) we should parameterize the image and checksum (or calculate the latter)
-  # Change to sda if using BM
-  openstack baremetal node set $node --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack.qcow2 --instance-info image_checksum=2a38fafe0b9465937955e4d054b8db3a --instance-info root_gb=25 --property root_device='{"name": "/dev/vda"}'
+  openstack baremetal node set $node --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack.qcow2 --instance-info image_checksum=2a38fafe0b9465937955e4d054b8db3a --instance-info root_gb=25  --property root_device="{\"name\": $ROOT_DISK}"
```

Comment:
common.sh needs to be sourced at the top of this file

Also it looks like your missing double quotes around ROOT_DISK to make it valid json